service/dap/daptest/client.go

```diff
@@ -267,6 +268,26 @@ func (c *Client) SetHitConditionalBreakpointsRequest(file string, lines []int, c
 	c.send(request)
 }
 
+// SetLogpointsRequest sends a 'setBreakpoints' request with logMessages.
+func (c *Client) SetLogpointsRequest(file string, lines []int, logMessages map[int]string) {
```

Comment:
I was trying to use the same naming scheme as SetConditionalBreakpointsRequest.

Maybe it would make more sense to have a single SetBreakpointsRequest(), that takes conditions, hit conditions, and log messages, since these are all now implemented.

Reply:
Yes, I was thinking that as well. That would remove all the duplicate code too, which is good.